frontend/src/stimulus/controllers/dynamic/project-storage-form.controller.ts

```diff
@@ -98,6 +98,7 @@ export default class ProjectStorageFormController extends Controller {
   selectProjectFolder(_evt:Event):void {
     const locals = {
       projectFolderHref: this.projectFolderHref,
+      createFolderHref: `${this.storage._links.self.href}/folders`,
```

Comment:
What I understand: This local was added to the place where creating a folder worked, but it wasn't added here and therefore adding a folder from the picker didn't work here.

Correct?

Reply:
full points, correct deduction.